src/sdk.ts

```diff
@@ -481,6 +483,34 @@ s	 * @example
 		});
 	}
 
+	/**
+	 * Get subscription modifiers.
+	 *
+	 * API documentation: https://developer.paddle.com/api-reference/f575ab89eb18c-list-modifiers
+	 *
+	 * @example
+	 * const result = await client.getSubscriptionModifiers();
+	 * const result = await client.getSubscriptionModifiers({ subscriptionID: 123 });
+	 */
+	getSubscriptionModifiers(options?: {
+		subscriptionID?: string;
+		planID?: string;
```

Comment:
Let's accept these only as numbers and convert them to strings in the body below.

Most likely these values will come from other API calls which return them as numbers.

Reply:
Done